main/webapp/modules/core/scripts/preferences.js

```diff
@@ -32,6 +32,7 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
 var preferenceUIs = [];
+var thePreferences;
```

Comment:
It looks like we can probably revert all changes in this file, so that this pull request does not touch this file at all, no?